docs/manual/guide/production/StandAloneService.md

```diff
@@ -0,0 +1,100 @@
+# How to run a Lagom service standalone
+
+We know that Lagom doesn't prescribe any particular environment, althougth Lagom provide out of the box ConductR, an integrated environment that provide several services to deploy and manage your services, it would be interesting, in some cases, to be able to deploy a service in standalone fashion.
+
+We know that one microservice cannot live without service locator. In development environment, Lagom's plugin provided one service locator out of the box. It exclusively dedicated to the *dev mode*.  The unique thing required to run a Lagom microservice in standalone mode is a service Locator.
+
+Here we will see how run a Lagom service in standalone mode.
+
+*To follow this guide, this imply you have one Zookeeper instance accessible.*
+
+As service locator registry, we will used Zookeeper. For that, we will use the [James Boner's project](https://github.com/jboner/lagom-service-locator-zookeeper) that implements service Locator interface of Lagom for Zookeeper.
+
+This guide will explain how to :
+
+* integrate the zookeeper service locator in your Lagom's project
+* package a service to run it in standalone manner
+
+We will use the following service implementation :
+
+
+@[content](code/docs/production/HelloWorldService.java)
+
+@[content](code/docs/production/HelloWorldServiceImpl.java)
+
+
+## How integrate the zookeeper service locator in your Lagom's project
+
+First step will be to add an implementation of the zookeeper service locator into your Lagom service.
+For that, we have to build locally the lagom-zookeeper-service-locator project.
+We have to do that because the project is under snapshot version.
+Well, clone the project with the command below :
```

Comment:
We use GFM which outputs linebreaks when linebreaks appear in the markdown source - these sentences should all be on one line.  Also "Well, ..." is a bit informal for documentation.